lib/snapshot/BUILD.gn

```diff
@@ -201,7 +201,7 @@ action("create_arm_gen_snapshot") {
   output_dir = "$root_out_dir/clang_x64"
   script = "//flutter/sky/tools/create_macos_gen_snapshots.py"
   visibility = [ ":*" ]
-  deps = [ "//third_party/dart/runtime/bin:gen_snapshot" ]
+  deps = [ "//third_party/dart/runtime/bin:gen_snapshot(//build/toolchain/$host_os:clang_x64)" ]
```

Comment:
Why not `$host_toolchain`? This will probably fail for custom toolchains.

Reply:
I did not know about `$host_toolchain` thanks! Fixed.